src/examples/learn_quant/meaning.py

```diff
@@ -0,0 +1,33 @@
+import pandas as pd
+from learn_quant.quantifier import QuantifierModel, QuantifierUniverse
+from itertools import product, combinations_with_replacement, permutations
+from ultk.language.sampling import powerset
+import random
+import argparse
+
+def create_universe(M_SIZE, X_SIZE):
+
+    possible_quantifiers = []
+
+    for x in combinations_with_replacement([0,1,2,3], r=M_SIZE):
+        combo = list(x) + [4] * (X_SIZE-M_SIZE)
+        permutations_for_combo = set(permutations(combo, r=len(combo)))
+        possible_quantifiers.extend(permutations_for_combo)
+    possible_quantifiers_name = set(["".join([str(j) for j in i]) for i in possible_quantifiers])
+
+    quantifier_models = set()
+    for name in possible_quantifiers_name:
+        quantifier_models.add(QuantifierModel(name=name))
```

Comment:
as per Nathaniel's recent commit, this should be a tuple not a set, so maybe these three lines can be shrunk to `quantifier_models = tuple(QuantifierModel(name=name) for name in possible_models_name)`